lock-keeper-tests/src/database.rs

```
@@ -87,6 +92,36 @@ impl TestDatabase {
         Ok(())
     }
 
+    /// Create an export key for testing using random bytes.
+    fn create_test_export_key(rng: &mut StdRng) -> OpaqueExportKey {
+        let mut key = [0_u8; 64];
+        rng.try_fill(&mut key)
+            .expect("Failed to generate random key");
```

I think this should return a result so it doesn't get swallowed up by our custom panic handler.